src/Sarif.Multitool.FunctionalTests/Update-Expected.ps1

```diff
@@ -1,28 +1,38 @@
 ﻿<#
 .SYNOPSIS
-Update the expected output files for the SarifCli functional tests.
+Update the expected output files for the Sarif.Multitool functional tests.
 
 .DESCRIPTION
-This script runs "SarifCli validate" on each input log file in the SarifCli
+This script runs "Sarif.Multitool validate" on each input log file in the SarifCli
 functional tests TestData directory. For each file foo.sarif, it produces
 an output file foo_Expected.sarif, overwriting any existing file of that
```

Comment:
>foo_Expected [](start = 15, length = 12)

fix it here, too. this term does not meet geopol standards

Reply:
Fixed here too.

---
In reply to: [205215240](https://github.com/Microsoft/sarif-sdk/pull/968#discussion_r205215240) [](ancestors = 205215240)